src/axolotl/cli/merge_lora.py

```diff
@@ -18,7 +18,15 @@ def do_cli(config: Path = Path("examples/"), **kwargs):
         return_remaining_strings=True
     )
     parsed_cli_args.merge_lora = True
-    parsed_cfg = load_cfg(config, merge_lora=True, **kwargs)
+
+    parsed_cfg = load_cfg(
+        config,
+        merge_lora=True,
+        load_in_8bit=False,
+        load_in_4bit=False,
+        flash_attention=False,
```

Comment:
What's the thing about `flash_attention` and LorA merges?

Reply:
Nevermind I just read [this comment](https://github.com/OpenAccess-AI-Collective/axolotl/issues/1002#issuecomment-1870358425) which explains everything